cmd/common.go

```diff
@@ -154,3 +169,51 @@ func prettyPrint(r *check.Controls, summary check.Summary) {
 		summary.Pass, summary.Fail, summary.Warn,
 	)
 }
+
+func savePgsql(jsonInfo	 string) {
+	envVars := map[string]string{
+		"PGSQL_HOST":     viper.GetString("PGSQL_HOST"),
+		"PGSQL_USER":     viper.GetString("PGSQL_USER"),
+		"PGSQL_DBNAME":   viper.GetString("PGSQL_DBNAME"),
+		"PGSQL_SSLMODE":  viper.GetString("PGSQL_SSLMODE"),
+		"PGSQL_PASSWORD": viper.GetString("PGSQL_PASSWORD"),
+	}
+
+	for k, v := range envVars {
+		if v == "" {
+			err := fmt.Errorf("Environment variable %s missing", envVarsPrefix+"_"+k)
+			panic(err)
```

Comment:
Please could you use the exitWithError() function rather than panic()? (Here and a couple more instances in the same function.)

Reply:
yep